modules/effects/arrow-data-instances-effects/gradle.properties

```diff
@@ -0,0 +1,4 @@
+# Maven publishing configuration
+POM_NAME=Arrow Type Classes Effects Instances for data types in Data
```

Comment:
:D the fact that I can't describe this properly with human words could highlight the "smell" on the approach. Anyways don't wanna get into the rabbit hole more than what I already did, so keeping it for now.

Reply:
"Instances of arrow-effects for the datatypes in arrow-data"

Reply:
Done